inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -497,8 +501,9 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
 
         auto fuse_activation_f = [&](activation_node& activation_node) {
             auto& input_data = activation_node.get_dependency(0);
-            if (input_data.get_users().size() != 1 || activation_node.get_dependencies().size() >= 3)
+            if ((input_data.get_users().size() != 1 && !input_data.has_fused_primitives()) || activation_node.get_dependencies().size() >= 3) {
```

Comment:
is this "!input_data.has_fused_primitives()" needed?
what about 
conv - elt---- elt    ? 
    \- act ----|

Reply:
Yes, that is correct. I change the code to support this case